ruby/README.md

```diff
@@ -23,44 +23,49 @@ bundle
 # initializing Link, see plaid.com/docs/faq/#does-plaid-support-international-bank-accounts-
 # for a complete list
 
-PLAID_CLIENT_ID='CLIENT_ID' \
-PLAID_SECRET='SECRET' \
-PLAID_PUBLIC_KEY='PUBLIC_KEY' \
-PLAID_ENV='sandbox' \
-PLAID_PRODUCTS='transactions' \
+PLAID_CLIENT_ID=[CLIENT_ID] \
+PLAID_SECRET=[SECRET] \
+PLAID_PUBLIC_KEY=[PUBLIC_KEY] \
+PLAID_PRODUCTS=[PRODUCTS] \
 PLAID_COUNTRY_CODES='US' \
-ruby app.rb
 
-# Go to http://localhost:4567
+bundle exec ruby app.rb
+# Go to http://localhost:8000
 ```
 
+### Link token creation and server-side configuration.
```

Comment:
nit: I think we should add a link to the docs here again.

`https://plaid.com/docs/#create-link-token`